scipy/optimize/_root.py

```diff
@@ -637,3 +639,169 @@ def _root_krylov_doc():
             See `scipy.sparse.linalg.lgmres` for details.
     """
     pass
+
+
+_iter = 100
+_xtol = 1e-12
+_rtol = np.finfo(float).eps * 2
```

Comment:
what's the point of these declarations over simply setting default values of `root_scalar` parameters?